src/7-to-8/major-changes/continuing-c7-c8.rst

```diff
@@ -0,0 +1,46 @@
+.. _compat_continuing_c7_with_c8:
+
+Continuing a Cylc 7 Workflow with Cylc 8
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. admonition:: Does This Change Affect Me?
+   :class: tip
+
+   Read this if you have a partially complete Cylc 7 workflow that you want to
+   continue, rather than start from scratch, with Cylc 8. Some cycling
+   workflows, for example, may need to run expensive "cold start" tasks and
+   incur a multi-cycle spin-up if started from scratch.
+
+.. warning::
+
+   Cylc 8 cannot restart a Cylc 7 workflow in-place, and continuing in a new
+   run directory involves some careful set up (below). So, **if possible you
+   should complete the run with Cylc 7**.
+
+
+To continue a Cylc 7 workflow with Cylc 8:
+
+1. Stop the Cylc 7 workflow at an convenient place
+
+   - Typically the end of a cycle point, to simplify the continuation
+2. :ref:`Install <MajorChangesInstall>` a new instance of the workflow from
+   source, with Cylc 8
+
+   - Adapt file paths to the new run directory structure, in workflow and task
+     configurations
+   - Note Cylc 8 does :ref:`remote file installation <728.remote-install>`
+     when a job is first submitted to a platform
+3. Copy runtime files needed by upcoming tasks from the old to the new run
+   directory
```

Comment:
This is ok for local hosts, however, on remote hosts it's trickier due to remote installation.

I think ATM we allow remote-installation to install "over the top of" an existing run dir? However, there's an issue open to change that which would require this step to occur *after* remote installation.

Maybe mention `work/` and `share/`?

Reply:
I've made two fairly open-ended comments that are relevant:
> - Note Cylc 8 does :ref:`remote file installation <728.remote-install>`
     when a job is first submitted to a platform
> - Note different files could be present on different job platforms

But OK, I'll be a bit more explicit, on those points. (Later...)

Reply:
(Just saw I haven't addressed this, and I've run out of time today ... but the existing text is not actually wrong, so I can do a quick follow-up PR later if this gets merged first).